constants.go

```diff
@@ -2,7 +2,7 @@ package libvirt
 
 const (
 	DriverName    = "libvirt"
-	DriverVersion = "0.9.1"
+	DriverVersion = "0.9.2"
```

Comment:
actually, it is better if we jump to `0.11`, as te original driver is `0.10` IIRC